srcpkgs/mpv/template

```diff
@@ -1,47 +1,47 @@
 # Template file for 'mpv'
 pkgname=mpv
-version=0.29.1
-revision=7
+version=0.30.0
+revision=1
 build_style=waf
 configure_args="--confdir=/etc/mpv --docdir=/usr/share/examples/mpv
- --enable-dvdread --enable-dvdnav --enable-cdda --enable-libmpv-shared
+ --enable-dvdnav --enable-dvdnav --enable-cdda --enable-libmpv-shared
  --enable-dvbin --enable-libarchive $(vopt_enable alsa) $(vopt_enable caca)
  $(vopt_enable jack) $(vopt_enable lua) $(vopt_enable oss oss-audio)
  $(vopt_enable pulseaudio pulse) $(vopt_enable sdl2)
- $(vopt_enable smb libsmbclient) $(vopt_enable sndio) $(vopt_enable v4l2 tv)
+ $(vopt_enable smb libsmbclient) $(vopt_enable sndio)
  $(vopt_enable vapoursynth) $(vopt_enable vdpau) $(vopt_enable wayland)
  $(vopt_enable x11) $(vopt_enable vulkan) $(vopt_enable vulkan shaderc)"
 hostmakedepends="pkg-config python-docutils perl $(vopt_if wayland wayland-devel)"
 makedepends="MesaLib-devel ffmpeg-devel harfbuzz-devel lcms2-devel libXv-devel
  libass-devel libbluray-devel libcdio-paranoia-devel libdvdnav-devel
- libdvdread-devel libguess-devel libuuid-devel libva-glx-devel rubberband-devel
+ libdvdnav-devel libguess-devel libuuid-devel libva-glx-devel rubberband-devel
```

Comment:
duplicate value of `libdvdnav-devel`

Reply:
good catch :+1:

Reply:
If we'd change this to proper bash arrays we could detect them easily...

Reply:
and at a reasonable speed...

Reply:
Ah, thanks for the review... I actually did update the common/shlibs, but I forgot to include it in the  commit.  Thanks for cleaning up and merging the PR.